### PR TITLE
Simplify create automation from device dialog

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

CleanShot 2024-10-02 at 14 25 14@2x

CleanShot 2024-10-02 at 14 25 27@2x

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by matthiasdebaat** [comment]
May I suggest to change the text to:
Use device as trigger
Use device as condition
Use device as action

And move "New automation" to the bottom.

File: src/translations/en.json [hunk 1]
```diff
@@ -4061,7 +4071,15 @@
...
             "no_scripts": "No scripts",
             "create": "Create script with {type}",
-            "create_disable": "Can't create script with disabled {type}"
+            "create_disable": "Can't create script with disabled {type}",
+            "new": {
+              "title": "New script",
+              "description": "Start with an empty script from scratch"
+            },
+            "actions": {
+              "title": "Script using the device as a condition",
```
Comment (bramkragten) [inline src/translations/en.json:4080]: this is action, not condition

File: src/translations/en.json [hunk 2]
```diff
@@ -4039,17 +4039,27 @@
...
             },
             "conditions": {
+              "title": "Automation using the device as a condition",
+              "description": "Only if a condition is met for the device",
               "caption": "Only do something if…",
               "no_conditions": "No conditions",
               "unknown_condition": "Unknown condition"
             },
             "actions": {
+              "title": "Automation using the device as a condition",
```
Comment (bramkragten) [inline src/translations/en.json:4061]: action, not condition

**Comment by piitaya** [comment]
> May I suggest to change the text to: Use device as trigger Use device as condition Use device as action
> 
> And move "New automation" to the bottom.

Done ! 🙂

**Review by bramkragten — APPROVED** [review]

**Pull request merged**